airtime_mvc/application/views/scripts/playlist/update.phtml

```diff
@@ -2,6 +2,35 @@
 $items = $this->contents;
 $isStaticSmartBlock = ($this->obj instanceof Application_Model_Block && $this->obj->isStatic());
 $isPlaylist = ($this->obj instanceof Application_Model_Playlist);
+if ($this->poolCount) { ?>
+<div class='sp_text_font sp_text_font_bold'>
+                <span id='sp_pool_count' class='sp_text_font sp_text_font_bold'>
+                <?php
+                if ($this->poolCount > 1) {
+                echo $this->poolCount;
+                ?>
+                <?php echo _("files meet the criteria")?>
+                    </span>
+    <span class='checked-icon sp-checked-icon' id='sp_pool_count_icon'></span>
+    <?php
+    } else if ($this->poolCount == 1) {
```

Comment:
You might want to consider using [plural support in zf1](https://framework.zend.com/manual/1.11/en/zend.translate.plurals.html) for this. You should be able to use the adapter as follows:

```php
echo $this->getTranslator()->plural("file meets the criteria", "files meets the criteria", $this->poolCount);
```

This would also shift the burden of knowing how to handle the zero case to the translators (TIL that not all languages handle the zero case like English).

Reply:
Looking into this. Your suggestion didn't work as getTranslator doesn't exist as a function. I'm trying to figure out if I should create a new Translator object to pull in plurals support. The only mention of Zend_Translate I see is in the rest RouteController.php file at this point.